hedera-mirror-rest/__tests__/integrationDbOps.js

```
@@ -189,11 +268,29 @@ const toAccount = function(str) {
 };
 
 const addAccount = async function(account) {
+  account = Object.assign({entity_shard: 0, entity_realm: 0, exp_time_ns: null}, account);
+
+  let e = accountEntityIds[account.entity_num];
```

if there's an existing test case where we add duplicate accounts for testing some corner case, this is okay.
If we don't have a reason to accept duplicates here, we should fail explicitly.
okay in followup.

---

Will look into it